.github/workflows/node-publish.yml

```diff
@@ -0,0 +1,41 @@
+name: 'DHIS2: Release'
+
+on:
+    push:
+        branches:
+            - master
+
+jobs:
+    process:
+        name: Publish
+        runs-on: ubuntu-latest
+        if: "!contains(github.event.head_commit.message, '[skip ci]')"
+        steps:
+            - uses: actions/checkout@v1
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: 12.x
+
+            - name: Install
+              run: yarn install --frozen-lockfile
+
+            - name: Lint
+              run: yarn lint
+
+            - name: Test
+              run: yarn test
```

Comment:
We're duplicating our testing and linting here... any reason for that?

Reply:
The other workflows for testing and linting run in parallel, so unless we have them here, theoretically we might publish something that fails tests and linting.

We could remove them here if it's too paranoid.